diff -rupN root-original/build/rmkdepend/main.c root-new/build/rmkdepend/main.c
--- root-original/build/rmkdepend/main.c	2014-09-27 02:01:09.051749758 -0700
+++ root-new/build/rmkdepend/main.c	2014-09-27 02:07:48.819735917 -0700
@@ -61,7 +61,7 @@ extern int fchmod(int, int);
 # elif defined(__CYGWIN__)
 extern int fchmod(int, mode_t);
 #else
-extern int fchmod(int, int);
+
 # endif
 #endif
 
diff -rupN root-original/core/unix/src/TUnixSystem.cxx root-new/core/unix/src/TUnixSystem.cxx
--- root-original/core/unix/src/TUnixSystem.cxx	2014-09-27 02:01:09.307749749 -0700
+++ root-new/core/unix/src/TUnixSystem.cxx	2014-09-27 02:08:18.219734899 -0700
@@ -4882,8 +4882,8 @@ void *TUnixSystem::SearchUtmpEntry(int n
 
 //---- System, CPU and Memory info ---------------------------------------------
 
-#if defined(R__MACOSX)
 #include <sys/resource.h>
+#if defined(R__MACOSX)
 #include <mach/mach.h>
 #include <mach/mach_error.h>
 
Binary files root-original/.git/index and root-new/.git/index differ
